README.adoc

```diff
@@ -247,37 +303,22 @@ InventoryReadinessCheck.java
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java[]
 ----
 
-In the [hotspot=isHealthy file=0]`isHealthy()` method, 
+In the [hotspot=isHealthy file=2]`isHealthy()` method, 
 you report the `inventory` service as not ready if the service is in maintenance or if its dependant service is unavailable.
 
-For simplicity, the custom [hotspot=3 file=2]`io_openliberty_guides_inventory_inMaintenance`
-MicroProfile Config property, which is defined in the [hotspot file=2]`resources/CustomConfigSource.json`
+For simplicity, the custom [hotspot=3 file=3]`io_openliberty_guides_inventory_inMaintenance`
```

Comment:
edit suggestions

For simplicity, the custom [hotspot=3 file=3]`io_openliberty_guides_inventory_inMaintenance`
MicroProfile Config property, that is defined in the [hotspot file=3]`resources/CustomConfigSource.json`
file, is used to indicate whether the service is in maintenance.
--->
For simplicity, the custom [hotspot=3 file=3]`io_openliberty_guides_inventory_inMaintenance`
MicroProfile Config property, which is defined in the [hotspot file=3]`resources/CustomConfigSource.json`
file,  indicates whether the service is in maintenance.